README.md

```diff
@@ -27,21 +41,22 @@ The Extensions API lets you do more without leaving Tableau. Build Tableau exten
 
    **npm start**
 
-5. Launch Tableau and try the sample extensions in a dashboard. The samples are located in the `Samples` folder.
+5. Launch Tableau and try a dashboard sample extension in a dashboard, or a viz extension in a worksheet. The dashboard and viz extension  samples are located in the `Samples` folder.
 
- >**Note** The local web server you start just serves to host the extension samples and extensions used in the tutorial, which have URLs similar to the following: `http://localhost:8765/Samples/DataSources/datasources.html` or `http://localhost:8765/Samples-Typescript/DataSources/datasources.html`
- >   This local web server is not intended to serve the Extensions API Help pages. 
+ >**Note** The local web server you start just serves to host the extension samples and dashboard extensions used in the tutorial. These extensions have URLs similar to the following: `http://localhost:8765/Samples/Dashboard/DataSources/datasources.html`.
+ >   This local web server is not intended to serve the Extensions API Help pages.
  >   View the Help on GitHub at [https://tableau.github.io/extensions-api](https://tableau.github.io/extensions-api).
 
 ### Typescript Development
+
 Samples written in Typescript are located in the `Samples-Typescript` folder.
 If you want to use TypeScript to write your extensions, you can run a script that starts up the HTTP server and actively listens for changes to the `.ts` files located in the `Samples-Typescript` folder. You can then add your extension to the folder and use the script to transpile your extension to JavaScript.
 
 * To start the the HTTP server and listen for changes to the `.ts` files.
 
     **npm run dev**
 
-For more information, see [Use TypeScript with the Extensions API](https://tableau.github.io/extensions-api/docs/trex_typescript.html).
+For more information, see [Use TypeScript with the Extensions API](https://tableau.github.io/extensions-api/docs/core/trex_typescript).
```

Comment:
I tried to click on this link and it couldnt find it. Not sure if that's fine for now.

Reply:
Right. Unfortunately, that link won't work until we switch to the Docusaurus build (switching to **GitHub Actions** instead of **Deploy from a Branch**).